Formula/powerman-dockerize.rb

```diff
@@ -0,0 +1,26 @@
+class PowermanDockerize < Formula
+  desc "Utility to simplify running applications in docker containers"
+  homepage "https://github.com/powerman/dockerize"
+  url "https://github.com/powerman/dockerize/archive/refs/tags/v0.16.0.tar.gz"
+  sha256 "b6507b24714dee193bd04be18fe8658531d2b0c0b9f6f060ac6bb387c736009f"
+  license "MIT"
+
+  depends_on "go" => :build
```

Comment:
Do you know if it will build with go 1.18? Otherwise we might want to use go@1.17 here

Reply:
I did not had a chance to test this yet, but it should works. If not - I'll fix this immediately.

Reply:
@SMillerDev, is the change to `depends_on "go@1.18" => :build` already possible? Seems like that 1.18 is not yet integrated into Homebrew.

Reply:
If powerman thinks it'll work I'm fine merging it with `depends_on "go" => :build`

Reply:
Okay, changed it back to `depends_on "go" => :build`.